crates/youki/Cargo.toml

```diff
@@ -31,3 +32,6 @@ clap_generate = { version = "3.0.0-beta.5" }
 [dev-dependencies]
 serial_test = "0.5.1"
 
+[build-dependencies]
+anyhow = "1.0"
```

Comment:
```suggestion
anyhow = "1.0.51"
```

Reply:
Bumped this in `[build-dependencies]` and also bumped `anyhow` in the `[dependencies]` section.